README.md

```diff
@@ -1,29 +1,570 @@
+        "image": {"@id": "http://schema.org/image", "@type": "@id"},
+        "Person": "http://schema.org/Person"
+    },
+    "@type": "Person"
+}
+```
+
+And we use it like this:
+```csharp
+var doc = JObject.Parse(_docJson);
+var frame = JObject.Parse(_frameJson);
+var opts = new JsonLdOptions();
+var flattened = JsonLdProcessor.Frame(doc, frame, opts);
+Console.WriteLine(flattened);
+
+/*
+
+Output:
+{
+    "@context": . . .,
+    "@graph": [
+        {
+            "@id": "_:b0",
+            "@type": "Person",
+            "image": "http://manu.sporny.org/images/manu.png",
+            "name": "Manu Sporny",
+            "homepage": "http://manu.sporny.org/"
+        }
+    ]
+}
+
+*/
+```
+
+
+### <a name="normalize"></a>[normalize](http://json-ld.github.io/normalization/spec/) (aka canonize)
+
+Normalized is a graph of objects that is a canonical representation of the document that can be used for hashing, comparison, etc.
+
+```csharp
+var doc = JObject.Parse(_docJson);
+var opts = new JsonLdOptions();
+var normalized = (RDFDataset)JsonLdProcessor.Normalize(doc, opts);
+Console.WriteLine(normalized.Dump());
+
+/*
+
+Output:
+@default
+    subject
+            type      blank node
+            value     _:c14n0
+    predicate
+            type      IRI
+            value     http://schema.org/image
+    object
+            type      IRI
+            value     http://manu.sporny.org/images/manu.png
+    ---
+    subject
+            type      blank node
+            value     _:c14n0
+    predicate
+            type      IRI
+            value     http://schema.org/name
+    object
+            type      literal
+            value     Manu Sporny
+            datatype  http://www.w3.org/2001/XMLSchema#string
+    ---
+    subject
+            type      blank node
+            value     _:c14n0
+    predicate
+            type      IRI
+            value     http://schema.org/url
+    object
+            type      IRI
+            value     http://manu.sporny.org/
+    ---
+    subject
+            type      blank node
+            value     _:c14n0
+    predicate
+            type      IRI
+            value     http://www.w3.org/1999/02/22-rdf-syntax-ns#type
+    object
+            type      IRI
+            value     http://schema.org/Person
+    ---
+    subject
+            type      IRI
+            value     http://example.org/ld-experts
+    predicate
+            type      IRI
+            value     http://schema.org/member
+    object
+            type      blank node
+            value     _:c14n0
+    ---
+    subject
+            type      IRI
+            value     http://example.org/ld-experts
+    predicate
+            type      IRI
+            value     http://schema.org/name
+    object
+            type      literal
+            value     LD Experts
+            datatype  http://www.w3.org/2001/XMLSchema#string
+    ---
+
+*/
+```
+
+
+### <a name="tordf"></a>ToRDF (N-Quads)
```

Comment:
Explicit anchors shouldn't be needed. Let's just use simpler titles instead, so the auto-generated anchors are predictable. Moving `(NQuads)` down as a paragraph means we need a short description of what it is.

```suggestion
### ToRDF

 (N-Quads)
```